karma.conf.cjs

```diff
@@ -1,49 +1,58 @@
 const webpack = require("webpack");
-const playwright = require('playwright');
+const playwright = require("playwright");
 
 process.env.CHROME_BIN = playwright.chromium.executablePath();
 process.env.FIREFOX_BIN = playwright.firefox.executablePath();
 
+const path = require("path");
+const tsConfigFile = path.resolve(__dirname, "./tsconfig.dev.json");
+
 module.exports = function (config) {
   config.set({
     frameworks: ["webpack", "mocha"],
     files: ["src/**/!(node).spec.ts"],
     preprocessors: {
-      "src/**/!(node).spec.ts": ["webpack"]
+      "src/**/!(node).spec.ts": ["webpack"],
     },
     envPreprocessor: ["CI"],
     reporters: ["progress"],
     browsers: ["ChromeHeadless", "FirefoxHeadless"],
     singleRun: true,
     client: {
       mocha: {
-        timeout: 6000 // Default is 2s
-      }
+        timeout: 6000, // Default is 2s
+      },
     },
     webpack: {
       mode: "development",
       module: {
-        rules: [{ test: /\.([cm]?ts|tsx)$/, loader: "ts-loader" }]
+        rules: [
+          {
+            test: /\.([cm]?ts|tsx)$/,
+            loader: "ts-loader",
+            options: { configFile: tsConfigFile },
```

Comment:
This change makes tests running in with `tsconfig.dev.json` which makes sense as we do have a lower bar for linting tests.

@waku-org/js-waku-developers